tests/test_edgeql_functions.py

```diff
@@ -7476,3 +7478,77 @@ async def test_edgeql_functions_multirange_unpack_01(self):
                 ],
                 json_only=True,
             )
+
+    async def test_edgeql_functions_encoding_base64_fuzz(self):
```

Comment:
Please add a test to see how gracefully it fails if you try to decode a non-base64 garbage. I'm curious if stray NULLs can appear or unexpected errors.

Reply:
Also test passing an empty string.

Reply:
Empty strings are already tested: `random.randrange(0, 1000)`

Reply:
Done.